torch/csrc/jit/passes/quantization.cpp

```diff
@@ -981,19 +1023,19 @@ graph(%self, %a_dequant, %w_scale, %w_zero_point, %w_dtype):
       auto packed_params_val = match_vmap.at(vmap.at("packed_params"));
 
       WithInsertPoint ins(packed_params_val->node());
-      // wrapper_module = self._packed_linear_weight_bias
+      // wrapper_module = self._packed_weight_bias
       Value* packed_params_module =
-          graph->insertGetAttr(graph->inputs()[0], "_packed_linear_weight_bias")
-              ->setType(wrapper.type());
+          graph->insertGetAttr(graph->inputs()[0], "_packed_weight_bias")
+              ->setType(wrapper_module.type());
 
       // packed_params = wrapper_module._packed_params
       Value* packed_params_from_attr =
           graph->insertGetAttr(packed_params_module, "_packed_params");
       packed_params_val->replaceAllUsesWith(packed_params_from_attr);
 
       // Delete nodes
-      std::vector<Node*> nodes_to_delete = {w_val->node(),
-                                            b_val->node(),
+      std::vector<Node*> nodes_to_delete = {//w_val->node(),
+                                            //b_val->node(),
```

Comment:
Why was this changed?

Reply:
I changed `%weight` and `%bias` to be the input of the pattern, when I try to delete them they have more uses because they are input of the method, so the uses is not empty for them and I can not delete them:
```
graph(%self : ClassType<Conv2d>,
      %input.1 : Tensor,
      %weight.1 : Tensor):
  %27 : float = prim::Constant[value=0.000733255]()
  %28 : int = prim::Constant[value=0]()
  %29 : int = prim::Constant[value=12]()
  %22 : float = prim::Constant[value=0.0783038]()
  %23 : int = prim::Constant[value=65]()
  %24 : int = prim::Constant[value=13]()
  %input.1.quant : Tensor = aten::quantize_per_tensor(%input.1, %22, %23, %24)
  %input.1.dequant : Tensor = aten::dequantize(%input.1.quant)
  %7 : None = prim::Constant() # /home/jerryzh/anaconda2/envs/ve3/lib/python3.6/site-packages/torch/nn/modules/conv.py:339:36
  %8 : int = prim::Constant[value=1]() # /home/jerryzh/anaconda2/envs/ve3/lib/python3.6/site-packages/torch/nn/modules/conv.py:336:46
  %9 : int[] = prim::Constant[value=[2, 2]]()
  %10 : int[] = prim::Constant[value=[3, 3]]()
  %11 : int[] = prim::Constant[value=[1, 1]]()
  %17 : float = prim::Constant[value=0.0275517]()
  %18 : int = prim::Constant[value=65]()
  %36 : ClassType<ConvPackedParams> = prim::GetAttr[name="_packed_weight_bias"](%self)
  %37 : Tensor = prim::GetAttr[name="_packed_params"](%36)
  %32 : Tensor, %33 : Tensor? = quantized::conv_unpack(%37)
  %34 : Tensor = aten::dequantize(%32)
  %35 : Tensor = aten::conv2d(%input.1.dequant, %34, %7, %9, %10, %11, %8)
  %19 : int = prim::Constant[value=13]()
  %12.quant : Tensor = aten::quantize_per_tensor(%35, %17, %18, %19)
  %12.dequant : Tensor = aten::dequantize(%12.quant)
  return (%12.dequant)
```